packages/mobile/src/verify/VerificationEducationScreen.tsx

```diff
@@ -152,7 +152,8 @@ function VerificationEducationScreen({ route, navigation }: Props) {
     navigation.setOptions({
       headerTitle: title,
       headerRight: () =>
-        !route.params?.hideOnboardingStep && (
+        !route.params?.hideOnboardingStep &&
+        account && (
```

Comment:
i think it is safe to do this? we never want to allow skipping this if there is no account created?